scholarly/_proxy_generator.py

```diff
@@ -396,6 +397,22 @@ def FreeProxies(self):
             if proxy_works:
                 break
 
+    def ScraperAPI(self, API_KEY):
+        """
+        Sets up a proxy using ScraperAPI
+
+        :Example::
+            pg = ProxyGenerator()
+            pg.ScraperAPI(API_KEY)
+
+        :param API_KEY: ScraperAPI API Key value. 
+        :type API_KEY: string
+        """
+        assert API_KEY is not None
+        self._ScraperAPI_KEY = API_KEY
+
+        self._use_proxy(http='http://api.scraperapi.com/?api_key='+API_KEY)
```

Comment:
What happens in the background if _use_proxy is not called?
Since the override happens directly in the navigator, I don't think this is needed.